src/commands/push-to-docker.yaml

```diff
@@ -0,0 +1,14 @@
+parameters:
+  image:
+    type: "string"
+  password:
+    type: "string"
+  registry:
+    type: "string"
+  username:
+    type: "string"
+steps:
+  - run: echo -n "<<parameters.image>>:$(architect project version)" > docker_image_name
+  - run: docker login --username "<<parameters.username>>" --password "<<parameters.password>>" "<<parameters.registry>>"
```

Comment:
Using password as parameter will print it to the CI output.

Reply:
I hope you didn't test that commit anywhere.

Reply:
Password wasn't printed to CI output, also using something like `echo -n <<parameters.password>>|docker login --password-stding` is not better when it comes to prevent password to be printed IMO. Not sure how we should handle this better.

Reply:
We are now using env variable names here, so value won't be exposed and we are more flexible.